src/prefect_server/api/runs.py

```diff
@@ -227,26 +174,39 @@ async def _create_flow_run(
         context=context or {},
         scheduled_start_time=scheduled_start_time,
         name=flow_run_name or names.generate_slug(2),
-        states=[
-            models.FlowRunState(
-                tenant_id=flow.tenant_id,
-                **models.FlowRunState.fields_from_state(
-                    Pending(message="Flow run created")
-                ),
-            )
-        ],
```

Comment:
No more pending state?

Reply:
We can't do the nested insert here because it will succeed even if the idempotency key is hit (due to the ON CONFLICT clause), so it's now done [here](https://github.com/PrefectHQ/server/pull/219/files#diff-aaf8abba6b80fc268ac079173dda255a41f8bd6ee121c9101229771497d63321R196). Since all flow runs (at this time!) go immediately into a scheduled state, there's no need for the intermediate pending one. In the future if we want to create unscheduled flow runs (for your backend `flow.run()` for example) then we can toggle out the `Scheduled` state for a `Pending` one.